### ISSUE TITLE
v1.1.15: Unable to serialize to JSON error

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#228]
While browsing my albums, this error popped up:
```
Logger: homeassistant.components.websocket_api.messages
Source: components/websocket_api/messages.py:197
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 20:07:02 (2 occurrences)
Last logged: 20:07:02

Unable to serialize to JSON. Bad data found at $.event.object_id=ProviderType.FILESYSTEM_LOCAL(<enum 'ProviderType'>
```

**Comment by marcelveldt** [comment]
Fix released in 1.1.17

**Comment by erkr** [comment]
Yep👍
